### ISSUE TITLE
達成方法集 F74の翻訳

### ISSUE THREAD

**Issue by yaytom-ba** [waic/wcag21#843]
対象ファイル：
https://github.com/waic/wcag21/tree/master/techniques/failures/F74.html
Techniques for WCAG 2.0にも同名のファイルがある文書です。

分割ワークフローを試験的に導入しています
作業の進め方は：Techniques for WCAG 2.1の作業の進め方（分割ワークフロー版）を参照ください。
https://github.com/waic/wcag21/blob/master/work-step_split.md

タイトルについて、表紙の目次ページ
https://github.com/waic/wcag21/blob/master/techniques/index.html
の該当箇所についても、編集を行ってください。
